docs/resources/sfs_turbo_dir.md

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "Scalable File Service (SFS)"
+---
+
+# huaweicloud_sfs_turbo_dir
+
+Provides an Shared File System (SFS) Turbo directory resource.
+
+## Example Usage
+
+### Create a STANDARD Shared File System (SFS) Turbo Directory
+
+```hcl
+variable "share_id" {}
+
+resource "huaweicloud_sfs_turbo_dir" "test" {
+  path     = "/tmp01"
+  share_id = var.share_id
+  mude     = 777
+  gid      = 100
+  uid      = 100
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the SFS Turbo directory resource. If omitted, the
+  provider-level region will be used. Changing this creates a new SFS Turbo directory resource.
+
+* `share_id` - (Required, String, ForceNew) Specifies the created file system ID. Changing this will create a new resource.
+
+* `path` - (Required, String, ForceNew) Specifies the valid full path of an existing directory. The parameter
+  starts with "/", otherwise the parameter is illegal. Changing this will create a new resource.
```

Comment:
I don't think it's necessary to require this path to exist, cause this resource is used to create a directory.